### PR TITLE
WIP: Feat/pre login page [TAS-89]

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

File: src/app/SiteHeader.tsx [hunk 1]
```diff
@@ -5,11 +5,19 @@ import { Header, HeaderLogged } from "@/components";
...
+const SiteHeader: React.FC<SiteHeaderProps> = ({ hasLogoImage }) => {
   const { data: session } = useSession();
   useThemeMode();
 
-  return session?.user ? <HeaderLogged /> : <Header />;
+  return session?.user ? (
+    <HeaderLogged hasLogoImage={hasLogoImage} />
+  ) : (
+    <Header hasLogoImage={hasLogoImage} />
+  );
```
Comment (renanpontez) [inline src/app/SiteHeader.tsx:20]:
```suggestion
  const HeaderComponent = session?user ? HeaderLogged : Header;
  
  return <HeaderComponent hasLogoImage={hasLogoImage} />
```

File: src/app/pre-login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+"use client";
```
Comment (renanpontez) [inline src/app/pre-login/page.tsx:1]: Can we remove this? I don't think we need this here, only inside FastLoginPage. 

Can you confirm @pedrohbl03?

File: src/components/atoms/NavBar/index.tsx [hunk 3]
```diff
@@ -1,22 +1,26 @@
 "use client";
```
Comment (renanpontez) [inline src/components/atoms/NavBar/index.tsx:1]: Do we also need this over here? As we're not using any client feature? I always get confused about this
@pedrohbl03

**Comment by femacedogomes** [comment]
u can see the animations in : https://deploy-preview-44--ez2live.netlify.app/pre-login

**Comment by femacedogomes** [comment]
WAITING a background image with better resolution.

**Pull request merged**